src/main/java/net/mcreator/io/OS.java

```diff
@@ -36,6 +36,15 @@ public static int getOS() {
 		return WINDOWS;
 	}
 
+	public static String getOSName() {

```

Comment:
Use existing getOS here so you don't re-do the os name matching logic again here